packages/frontend/src/components/Game/GameBoard/Tile/index.jsx

```diff
@@ -2,13 +2,14 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import './style.scss';
 
-const Tile = ({ tile, onFlip }) => {
+const Tile = ({ tile, onFlip, player }) => {
   const onClickTile = () => {
     onFlip(tile);
   };
 
   const tileClasses = [
-    tile.faceUp ? 'faceUp' : 'faceDown tile border border-primary',
+    tile.faceUp ? 'faceUp' : 'faceDown tile border',
+    player.number === 1 ? 'border-primary' : `border-${player.color.className}`,
```

Comment:
If the `activePlayer` were added to the context, this could be `border-${activePlayer.color.className}`